content/en/agent/troubleshooting/debug_mode.md

```diff
@@ -110,13 +110,15 @@ The following Agent log levels are available for `log_level` or `DD_LOG_LEVEL`:
 
 | Option     | Critical logs | Error logs | Warn logs | Info logs | Debug logs | Trace logs |
 |------------|---------------|------------|-----------|-----------|------------|------------|
-| `OFF`      |               |            |           |           |            |            |
-| `CRITICAL` | {{< X >}}     |            |           |           |            |            |
-| `ERROR`    | {{< X >}}     | {{< X >}}  |           |           |            |            |
-| `WARN`     | {{< X >}}     | {{< X >}}  | {{< X >}} |           |            |            |
-| `INFO`     | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} |            |            |
-| `DEBUG`    | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} | {{< X >}}  |            |
-| `TRACE`    | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} | {{< X >}}  | {{< X >}}  |
+| `'OFF'`      |               |            |           |           |            |            |
+| `'CRITICAL'` | {{< X >}}     |            |           |           |            |            |
+| `'ERROR'`    | {{< X >}}     | {{< X >}}  |           |           |            |            |
+| `'WARN'`     | {{< X >}}     | {{< X >}}  | {{< X >}} |           |            |            |
+| `'INFO'`     | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} |            |            |
+| `'DEBUG'`    | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} | {{< X >}}  |            |
+| `'TRACE'`    | {{< X >}}     | {{< X >}}  | {{< X >}} | {{< X >}} | {{< X >}}  | {{< X >}}  |
+
+**Note**: When setting the log level to `'OFF'` in the configuration file quotes are mandatory to prevent the value for being improperly parsed. Quotes are optional for other log levels.
```

Comment:
```suggestion
**Note**: When setting the log level to `'OFF'` in the Agent main configuration file, quotes are mandatory to prevent the value for being improperly parsed.
```